.github/workflows/functional-tests-imx.yml

```diff
@@ -1,7 +1,7 @@
-name: Functional Tests
+name: Functional Tests (IMX)
 on:
   push:
-    branches: [ main ]
+    branches: [ main, DX-2552/port-zkevm-tests ]
```

Comment:
what does this additional branch do here?

Reply:
I should have removed it. fixed.